### PR TITLE
core: rename preload-lcp-image to prioritize-lcp-image

### PR DESCRIPTION
**Pull request by brendankenny**
to better match the goal of the audit and to prepare for upcoming changes

see #13738

### CODE REVIEW

File: core/test/lib/stack-packs-test.js [hunk 1]
```diff
@@ -260,6 +260,7 @@ Array [
       Array [
         "unminified-warning",
         "disable-bundling",
+        "preload-lcp-image",
```
Comment (adamraine) [inline core/test/lib/stack-packs-test.js:263]: This is a bug right?

Comment (brendankenny) [reply core/test/lib/stack-packs-test.js:263]: it's because there are stack packs for `preload-lcp-image` so they move to `unrecognizedKeys`

Comment (brendankenny) [reply core/test/lib/stack-packs-test.js:263]: updated stack pack ids in https://github.com/GoogleChrome/lighthouse-stack-packs/pull/89

**Comment by connorjclark** [comment]
lighthouse-stack-packs@1.9.1 is available now

**Review by adamraine — APPROVED** [review]

**Pull request merged**